common/nns.go

```diff
@@ -0,0 +1,43 @@
+package common
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop"
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+	"github.com/nspcc-dev/neo-go/pkg/interop/convert"
+	"github.com/nspcc-dev/neo-go/pkg/interop/lib/address"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/management"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/std"
+)
+
+// NNSID is the ID of the NNS contract in NeoFS networks. It's always deployed
+// first.
+const NNSID = 1
+
+// ContractTLD is the default domain used by NeoFS contracts.
+const ContractTLD = "neofs"
+
+// InferNNSHash returns NNS contract hash by [NNSID] or panics if
+// it can't be resolved.
+func InferNNSHash() interop.Hash160 {
+	var nns = management.GetContractByID(1)
+	if nns == nil {
+		panic("no NNS contract")
+	}
+	return nns.Hash
+}
+
+// ResolveFSContract returns contract hash by name as registered in NNS or
+// panics if it can't be resolved.
+func ResolveFSContract(name string) interop.Hash160 {
```

Comment:
This one needs to be rebased, https://github.com/nspcc-dev/neofs-contract/pull/368 has a notion of DRY for this part of code.